### DISCUSSION TITLE
Welcome to dice discussions!

### DISCUSSION THREAD

**Discussion by MoraGames** [MoraGames/dices#2] [Announcements]
# 💬   Discussion Time!
## 👋  Welcome!
Discussion has been opened for this repository, allowing anyone to comment on their opinion of the repository.

Through this section it will also be possible to suggest future changes / implementations, which will be analyzed by the developers and, if approved, implemented as issues.

We thank all those who decide to participate, for any reason.

## 📜  Rules
- Don't be **insistent**, neither towards other users, nor towards developers.
- Don't be **vulgar**, **offensive** or **discriminatory** towards other users or developers.
- Don't **criticize** other users or developers unless they are constructive and well argued.
- Don't **spam** (sending messages for advertising purposes) unless authorized by an authoritative member.
- Don't **flood** (sending messages with a high frequency).

Note: _Any behavior that is logically or morally incorrect, even if it is not specifically indicated in a rule, is prohibited._

## 🥳  Enjoy!
Fear not, no one is here to judge others, get free and speak openly express your thoughts about the repository.
We are waiting for you, have fun!

## 👥   Collaborator
- @MoraGames `creator`, `moderator`, `developer`, `alpha tester`
- @KiritoNya `developer`, `alpha tester`
- @Vano2903 `alpha tester`,

Note: _Since the package will be publicly available, anyone can download it and play the role of `tester`.
We apologize for not being able to include all participants, but we appreciate your contribution and will keep a note of it for the more active!_

**Comment by Vano2903** [comment]
ciao xD
